### PR TITLE
chore: added forms page to connect website.

### PR DESCRIPTION
**Pull request by cooperwatson**
https://auctane.atlassian.net/browse/CAR-6574

Added a forms page to better describe Carrier Registration and Settings forms and Order Source Connection forms.

* Also a small description field fix for the customs page
* Added a carrier specific address page, and renamed references of addresses

### CODE REVIEW

**Comment by ricky** [comment]
Something wonky with the formatting on the forms page. The content is extending beyond the boundary.

Screen Shot 2021-11-03 at 2 56 59 PM

**Comment by cooperwatson** [comment]
> Something wonky with the formatting on the forms page. The content is extending beyond the boundary.
> 
> Screen Shot 2021-11-03 at 2 56 59 PM

Weird. My page is fine. Locally and in the vercel preview.
Is there an error in the console?

File: src/layouts/docs/menu-contents.ts [hunk 1]
```diff
@@ -376,6 +380,10 @@ export const menu: MenuContents = [
             "title": "Auth",
             "href": "/docs/reference/auth"
           },
+          {
+            "title": "Forms",
+            "href": "/docs/reference/forms"
```
Comment (ricky) [inline src/layouts/docs/menu-contents.ts:385]: We should probably include a link in the order app reference section as well

Comment (cooperwatson) [reply src/layouts/docs/menu-contents.ts:385]: It's in both. This line is even the order app one.
It's just the ui only highlights the first one in the menu when you're on the page which is the carrier app link.

Comment (ricky) [reply src/layouts/docs/menu-contents.ts:385]: Ahh, I was looking for it alphabetically on the page... 🤦

**Review by ricky — APPROVED** [review]

**Pull request merged**